main_test.go

```diff
@@ -35,39 +183,572 @@ func TestExample2(t *testing.T) {
+}
+
+func TestCreateNewOrderProductIdIsFloat(t *testing.T) {
+	err := createOrdersData([][]string{}, [][]string{}, [][]string{{"store", "create-order", "1.5", "4"}})
+	assert.ErrorContains(t, err, "Must be valid integer")
+}
+
+func TestCreateNewOrderWithValidInputs(t *testing.T) {
+	err := tearDownOrders()
+	assert.NilError(t, err)
+
+	db, err := connectDB()
+	assert.NilError(t, err)
+
+	err = createOrdersData([][]string{{"store", "create-customer", "vivek.s@outreach.io", "WA"}}, [][]string{{"store", "create-product", "--sku=abcde", "banana", "5"}}, [][]string{{"store", "create-order", "1", "1"}})
+
+	var orderID int
+	var createdAt sql.NullString
+	var customerID int
+	var productID int
+	var sku sql.NullString
+
+	rows := db.QueryRow("SELECT * FROM Orders ORDER BY ID DESC LIMIT 1")
+
+	assert.NilError(t, rows.Err())
+
+	err = rows.Scan(&orderID, &createdAt, &customerID, &productID, &sku)
+	assert.NilError(t, err)
+	assert.Equal(t, productID, 1)
+	assert.Equal(t, customerID, 1)
+	assert.Equal(t, orderID, 1)
+}
+
+func TestCreateNewOrder_ValidInputs_CustomerForeignKey_Error(t *testing.T) {
+	err := tearDownOrders()
+	assert.NilError(t, err)
+
+	err = createOrdersData([][]string{}, [][]string{{"store", "create-product", "--sku=abcde", "banana", "5"}}, [][]string{{"store", "create-order", "1", "1"}})
+
+	assert.ErrorContains(t, err, "Violates Foreign Key constraints")
+}
+
+func TestCreateNewOrder_ValidInputs_ProductForeignKey_Error(t *testing.T) {
+	err := tearDownOrders()
+	assert.NilError(t, err)
+
+	err = createOrdersData([][]string{{"store", "create-customer", "vivek.s@outreach.io", "WA"}}, [][]string{}, [][]string{{"store", "create-order", "1", "1"}})
+
+	assert.ErrorContains(t, err, "Violates Foreign Key constraints")
+}
+
+func Example_CreateOrder_WithCorrectOutput() {
+	printOrder(1, 1, 1)
+
+	//Output:
+	//OrderID    |ProductID    |CustomerID    |
+	//1          |1            |1             |
+}
+
+// ### We should have show commands for customers, products, and orders:
+
+// 1. Command syntax: `store show-orders [--name=<name>]`
+
+// This command should display all of the orders in the orders table.
+// This command should have an optional --name flag which allows users to filter by name.
+// This parameter should support wildcard matching (e.g. 'foo%')
+
+// Examples:
+
+//       > store show-products
+//       PRODUCT_ID        NAME       PRICE       SKU
+//       1                 Bananas     3.99       aa1
+//       2                 Milk        4.50
+//       3                 Cookies     1.99       ab2
+
+//       > store show-products --name "M%"
+//       PRODUCT_ID        NAME       PRICE       SKU
+//       2                 Milk        4.50
+//       5                 Markers     8.99       xi1
+
+// 1. test output of show-products no flag
+// 2. test output of show-products with name flag
+
+func Example_ShowProductsNoFlag() {
+	ctx, _ := context.WithTimeout(context.Background(), time.Second*30)
+
```

Comment:
Make sure to always include the `defer cancel()` call when using the `context` package (no need to dig into it too much - it's just good practice)

```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
	defer cancel()
```